app/components/ViewContainer.js

```diff
@@ -6,6 +6,22 @@ import PinList from './PinList';
 import Button from 'react-native-button';
 import { myCurrLoc } from '../lib/db/db';
 
+const styles = StyleSheet.create({
+  ViewMenu: {
+    position: 'absolute',
+    top: 25,
+    right: 25,
+    flexDirection: 'row',
+  },
+  ViewButton: {
+    paddingVertical: 10,
+    paddingHorizontal: 12,
+    color: '#fff',
+    fontWeight: 'bold',
+    backgroundColor: '#2e8b7d',
+  },
```

Comment:
Can we add a teeny tiny right ( or left ) margin to differentiate between the two buttons?